Multisig/UI/App/HeaderViewController/HeaderViewController.swift

```diff
@@ -78,7 +78,7 @@ final class HeaderViewController: ContainerViewController {
 
             if let safe = selectedSafe {
                 safeBarView.setAddress(safe.addressValue)
-                safeBarView.setName(safe.displayName)
+                safeBarView.setName(safe.displayName, readOnly: App.configuration.toggles.signing)
```

Comment:
What is the logic behind here? I think this toggle can be removed at all.
We need to check it an owner is imported for this safe. This might be tricky because we do not store safe owners, but not we need to store them and update periodically (probably on app start & safe settings screen loading).

We need to align on update logic with Android here - might be a good topic for requirements refinement meeting.